client/src/components/LoginDialog.tsx

```diff
@@ -93,12 +110,15 @@ export default function LoginDialog() {
   const [avatarIndex, setAvatarIndex] = useState<number>(0)
   const dispatch = useAppDispatch()
   const connected = useAppSelector((state) => state.user.connected)
+  const videoConnected = useAppSelector((state) => state.user.videoConnected)
+  const emptyNameField = useAppSelector((state) => state.user.emptyNameField)
```

Comment:
This is a component local state that does not share with other components. No need to store in redux store, use local state `useState` instead.

Also emptyNameField sounds like a verb, maybe `nameFieldError`, `nameFieldInvalid`, or `nameFieldEmpty`.

Reply:
Fixed!